.github/workflows/ci.yml

```diff
@@ -32,8 +34,14 @@ jobs:
           node --version
           npm --version
           npx commitlint --version
+      - name: Compile ts files
```

Comment:
@realmarv ts is not a blessed abbreviation in our list :) which means you probably meant to use `TS` (acronym) or `typescript` or `.ts`

Reply:
Done